js/src/model.ts

```diff
@@ -0,0 +1,80 @@
+import * as tf from '@tensorflow/tfjs';
+import * as tfn from '@tensorflow/tfjs-node';
```

Comment:
This import breaks the browser package, as `tfjs-node` is not available there. 
Since `tfn` is only used once,  can you import `tfjs-node` dynamically instead? 
Removing this line and adding something like `const tfn = await import('@tensorflow/tfjs-node')` on line 48  (+ some error checking) should fix this.

To test that it works properly, run:
```
cd js
npm run build
npm link
cd ../website
npm install
npm link 'magika'
npm run dev
```

Right now, you'd get this output:

```

  VITE v5.1.4  ready in 348 ms

  ➜  Local:   http://localhost:5173/magika/
  ➜  Network: use --host to expose
  ➜  press h + enter to show help
✘ [ERROR] No loader is configured for ".html" files: ../js/node_modules/@mapbox/node-pre-gyp/lib/util/nw-pre-gyp/index.html

    ../js/node_modules/@mapbox/node-pre-gyp/lib/node-pre-gyp.js:86:21:
      86 │       return require('./' + command)(self, argvx, callback);
         ╵                      ~~~~~~~~~~~~~~

✘ [ERROR] Could not resolve "mock-aws-s3"

    ../js/node_modules/@mapbox/node-pre-gyp/lib/util/s3_setup.js:43:28:
      43 │     const AWSMock = require('mock-aws-s3');
      [...]
 ```